docs/tools-reference/custom-kernels-distros/run-a-distributionsupplied-kernel-with-pvgrub.md

```diff
@@ -564,46 +564,58 @@ CentOS 5
+        mkdir /boot/grub
 
-        yum install kernel.x86_64
+4. Take a note of the file names in the `/boot` directory:
 
-3.  Create a file named `/boot/grub/menu.lst` with the following contents. Adjust the `title`, `kernel`, and `initrd` lines to reflect the actual file names found in the `/boot/` directory.
+        ls -1 /boot
+        
+        System.map-3.17.6-300.fc21.x86_64
+        config-3.17.6-300.fc21.x86_64
+        grub
+        initramfs-0-rescue-6b4f4bd432094ba4beb59d23c6d307ef.img
+        initramfs-0-rescue-fafe7c1120e04fda872218011b85367e.img
+        initramfs-3.17.6-300.fc21.x86_64.img
+        initrd-plymouth.img
+        vmlinuz-0-rescue-6b4f4bd432094ba4beb59d23c6d307ef
+        vmlinuz-0-rescue-fafe7c1120e04fda872218011b85367e
+        vmlinuz-3.17.6-300.fc21.x86_64
 
-    {: .file }
-	/boot/grub/menu.lst
-	: ~~~
-		timeout 5
-    	
-    	title Fedora 17, kernel 3.9.10-100.fc17.x86\_64 root (hd0) kernel /boot/vmlinuz-3.9.10-100.fc17.x86\_64 root=/dev/xvda ro quiet initrd /boot/initramfs-3.9.10-100.fc17.x86\_64.img
-	~~~
-	
-4.  In the Linode Manager, edit your Linode's configuration profile to use either **pv-grub-x86\_32** or **pv-grub-x86\_64** as the **Kernel**, depending on the version of Fedora you have deployed (32-bit or 64-bit).
-5.  Make sure the root device is specified as **xvda**.
-6.  Save your changes by clicking **Save Profile** at the bottom of the page.
-7.  Reboot your Linode from the **Dashboard** tab.
-8.  Once your Linode has rebooted, log in via SSH and issue the following command:
+5. Create a file named `/boot/grub/menu.lst` with the following contents, replacing values to match the contents of the `/boot` directory:
 
-        uname -a
+        timeout 5
+        title Fedora 21 (3.17.6-300.fc21.x86_64)
+        root (hd0)
+        kernel /boot/vmlinuz-3.17.6-300.fc21.x86_64 root=/dev/xvda
+        initrd /boot/initramfs-3.17.6-300.fc21.x86_64.img
 
-    You should see output similar to the following, indicating you're running the native kernel:
+6. In the Linode Manager, edit your Linode's Configuration Profile:
+
+    - Change the Kernel to `pv-grub-x86_64`
+    
+    - Make sure the root device is `/dev/xvda`
+    
+7. Save your profile and reboot your Linode
 
-        Linux li63-119 3.9.10-100.fc17.x86_64 #1 SMP Sun Jul 14 01:31:27 UTC 2013 x86_64 x86_64 x86_64 GNU/Linux
+8. Once your Linode has rebooted, you can confirm that your new kernel is running by logging in to ssh and issuing the following command:
 
-Note that if you later install an updated kernel, you'll need to add an entry for it to your `menu.lst` file. By default, the first kernel in the list will be booted.
+        uname -a
 
+   You should see output similar to the following:
 
+        Linux dz8 3.17.6-300.fc21.x86_64 #1 SMP Mon Dec 8 22:29:32 UTC 2014 x86_64 x86_64 x86_64 GNU/Linux
 
+Note that if you later install an updated kernel, you'll need to add an entry for it to your `/boot/grub/menu.lst` file.
```

Comment:
Most unfortunate.

Better would be to chain grub-xen, and avoid writing menu.lst entirely.